workload/plugins/custom-plugin-select-host-and-boot.py

```diff
@@ -0,0 +1,33 @@
+from rally.plugins.openstack import scenario
```

Comment:
I must to import this file it's useful for: https://github.com/BeyondTheClouds/enos/pull/141/files#diff-92e32cbb6ed56a30fea47aa5fcceefe8R31